test/terraform_aws_ssm_example_test.go

```diff
@@ -0,0 +1,42 @@
+package test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+)
+
+func TestTerraformAwsSsmExample(t *testing.T) {
+	t.Parallel()
+	region := "us-east-2"
```

Comment:
We don't usually hard-code regions. Instead, we pick a random one (using a Terratest helper) to ensure the code works in multiple regions.